build.sh

```diff
@@ -2,7 +2,7 @@
 
 usage()
 {
-    echo "Usage: $0 [managed] [native] [BuildArch] [BuildType] [clean] [verbose] [clangx.y] [platform] [cross] [skiptests] [cmakeargs]"
+    echo "Usage: $0 [managed] [native] [BuildArch] [BuildType] [clean] [verbose] [clangx.y] [platform] [cross] [skiptests] [staticcurl]  [cmakeargs]"
```

Comment:
Long term it feels like static linking should be an all or nothing propitiation (except maybe we always dynamically link to glibc).  Do you imagine that as we add patches to static link against more stuff, this switch becomes just "static"?

Would it make sense to do that now?  I'm torn.